src/boot/efi/meson.build

```diff
@@ -3,6 +3,10 @@
 efi_config_h_dir = meson.current_build_dir()
 efi_addon = ''
 
+if conf.get('ENABLE_BOOTLOADER') != 1
+        subdir_done()
+endif
```

Comment:
This will probably hide the fuzzers from CI jobs. Currently, the tests/fuzzers can be built without pyelftools around.

Reply:
Can't we just install pyelftools there? It needs python anyway to compile anything

Reply:
ok, added (hope tools/oss-fuzz is the right place, but it looks like it is)

Reply:
I don't see fuzz-efi- targets in cifuzz output. :crying_cat_face:

Apparently, it's running on focal, which is python 3.8.

Reply:
oh ffs

Reply:
dropped that from here, will open a different PR